action.yml

```diff
@@ -131,7 +131,7 @@ inputs:
     description: "Internal use only"
     required: false
   openai_api_key:
-    description: "The OpenAI API key to authenticate to access LLM suggestions"
+    description: "The OpenAI API key to authenticate to access LLM suggestions. If left empty it will use Speakeasy's platform key withing platform limits."
```

Comment:
```suggestion
    description: "The OpenAI API key to authenticate to access LLM suggestions. If left empty it will use Speakeasy's key within platform limits."
```